csrc/selective_scan/selective_scan_fwd_kernel.cuh

```diff
@@ -8,9 +8,14 @@
 #include <c10/util/Half.h>
 #include <c10/cuda/CUDAException.h>  // For C10_CUDA_CHECK and C10_CUDA_KERNEL_LAUNCH_CHECK
 
-#include <cub/block/block_load.cuh>
-#include <cub/block/block_store.cuh>
-#include <cub/block/block_scan.cuh>
+#ifndef USE_ROCM
+    #include <cub/block/block_load.cuh>
+    #include <cub/block/block_store.cuh>
+    #include <cub/block/block_scan.cuh>
+#else
+    #include <hipcub/hipcub.hpp>
+    namespace cub = hipcub;
+#endif
```

Comment:
Have we not added these cuda to hip mappings yet in pytorch?

Reply:
Yes, the same comment applies here as with the bf16 include. We would appreciate your advice on how to handle backward compatibility & on pytorch versioning. I.e. how soon the changes to "cuda_to_hip_mappings.py" are incorporated into, say, pytorch nightly.